generic3g/OuterMetaComponent/write_restart.F90

```diff
@@ -30,6 +30,7 @@ module recursive subroutine write_restart(this, importState, exportState, clock,
          geom = this%get_geom()
          restart_handler = RestartHandler(name, geom, clock, _RC)
          call restart_handler%write("import", importState, _RC)
+         call restart_handler%write("export", exportState, _RC)
```

Comment:
Doesn't this need some sort of conditional?   We only want to write the export in the testing context.

Reply:
@tclune Added a config key `write_exports`. When that is set to `true`, exports are checkpointed.